medusa/providers/generic_provider.py

```diff
@@ -493,15 +493,20 @@ def parse_pubdate(pubdate, human_time=False, timezone=None):
 
         :returns: a datetime object or None
         """
+        now_alias = ['right now', 'just now', 'Now']
```

Comment:
Probably it would be better to have all of them lowercase and use re.I to ignore case in the regex. This could be a tuple instead of a list as well